### DISCUSSION TITLE
Change structure of thumbnail folder

### DISCUSSION THREAD

**Discussion by saosangmo** [Studio-42/elFinder#3357] [Q&A]
hello,
Because I want to improve I/O speed, so want to change the default thumnbail folders like the below tree:
```
Root
-Folder 1
-.tmb (of Folder 1)
---Folder 1.1
---.tmb (of Folder 1.1)
---Folder 1.2
---.tmb (of Folder 1.2)
-Folder 2
-.tmb (of Folder 2)
....

```
How I can do this?
thank you so much!

**Comment by nao-pon** [comment]
@saosangmo #1795 will be helpful.
